test-data/typecheck/test_config.yml

```diff
@@ -38,3 +38,16 @@
               class MyModel(models.Model):
                   user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
 
+-   case: add_mypy_path_to_package_search
```

Comment:
We can add a test for `MYPYPATH` as well.
`pytest-mypy-plugins` supports `env:` section.
see https://github.com/typeddjango/pytest-mypy-plugins/blob/master/pytest_mypy_plugins/tests/test-paths-from-env.yml

Reply:
I've added the test for MYPYPATH env var. Let me know if there is anything else I should add. Thanks for the help!